### DISCUSSION TITLE
如果正确配置 PDF 或 Epub 后仍无法正确导出文件的话请看这里的解决方案（因标题含有特殊字符造成的问题）

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#1617] [重要通知]
## 问题描述

RT

## 使用前提

**首先确保配置成功**，关于配置 PDF 的相关问题可以看 https://github.com/Kenshin/simpread/discussions/1562 https://github.com/Kenshin/simpread/discussions/1566 这两个说明

## 解决方案

猜测大概率是因为导出的标题含有特殊字符造成的，解决方案请安装这个插件 https://github.com/Kenshin/simpread/discussions/1603

## 注意事项

1. 解决方案仅能解决在阅读模式界面下导出 PDF 因为标题含有字符无法导出的问题。

2. 无法解决在稍后读中导出 PDF ，暂时可以使用（修改标题去掉特殊字符）的方式解决。（ 此问题会在简悦 · 同步助手 升级到下个版本后自行解决，具体请留意同步助手的更新说明）

**Comment by nachifur** [comment]
有用

**Reply by Kenshin** [reply]
恭喜「自救」成功，下个版本会把这个功能集成到扩展中，避免加入到稍后读存在类似的问题。 😀

**Reply by FOkvj** [reply]
翻墙才能发邮件或保存到笔记应用

**Reply by Kenshin** [reply]
发邮件不存在这个情况。
笔记的话，是跟笔记 API 有关系，除个别的外，也都不需要。

请确认你的网络环境。
